apps/marketplace/components/Brief/BriefDownloadResponses.js

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import AUheading from '@gov.au/headings/lib/js/react.js'
+import AUbutton from '@gov.au/buttons/lib/js/react.js'
+import AUpageAlert from '@gov.au/page-alerts/lib/js/react.js'
+import { rootPath } from 'marketplace/routes'
+import styles from './BriefDownloadResponses.scss'
+
+export class BriefDownloadResponses extends Component {
+  constructor(props) {
+    super(props)
+
+    this.handleButtonClick = this.handleButtonClick.bind(this)
+    this.getFileSizeAndType = this.getFileSizeAndType.bind(this)
+  }
+
+  getFileSizeAndType() {
+    const bytes = this.props.brief.responsesZipFilesize
+    let result = ''
+    if (this.props.brief.lot === 'digital-professionals') {
+      let size = ''
+      if (bytes < 1048576) {
+        size = `${parseFloat(bytes / 1024).toFixed(2)}KB`
+      } else {
+        size = `${parseFloat(bytes / 1024 / 1024).toFixed(2)}MB`
+      }
+      result = `${size} zip`
+    } else if (this.props.brief.lot === 'digital-outcome') {
+      result = 'csv'
+    }
+    return result
+  }
+
+  handleButtonClick(e) {
+    e.preventDefault()
+    window.location = `/api/2/brief/${this.props.brief.id}/respond/documents`
+  }
+
+  render() {
+    if (this.props.briefResponses.length === 0) {
```

Comment:
will this work for both specialists and outcomes?

Reply:
yes outcomes still populate the `briefResponses` in the API response.